flasher.cpp

```diff
@@ -188,15 +173,6 @@ void Flasher::LoopHandler()
     case FlasherStates::kCheckBoardInfo:
         if (CollectBoardId()) {
             emit ShowTextInBrowser("Board ID: " + board_id_);
-            if (GetBoardKey()) {
-                emit ShowTextInBrowser("Board verified!");
-                emit SetReadProtectionButtonText(IsFirmwareProtected());
-            }
-            else {
-                emit ShowTextInBrowser("Board is not registered! Press the register button");
-                emit EnableRegisterButton();
```

Comment:
Removed the Register button. Originally it was designed to catch keys from the server but since we removed the key it is also obsolete.  In preparation is a bit different client/server communication.